### DISCUSSION TITLE
tool issue

### DISCUSSION THREAD

**Discussion by blood607** [projectdiscovery/httpx#1956] [Q&A]
"Use with caution. You are responsible for your actions.
Developers assume no liability and are not responsible for any misuse or damage."
This is showing whenever I launched the tool.

**Comment by dogancanbakir** [comment]
Yes, this is the disclaimer we print with every run. What's the issue you're having? Could you provide a bit more detail? Like, cmd and the full output.

**Comment by ehsandeep** [comment]
@blood607 use `-silent` option if you don't want to see it.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
